modules/nw-udn-cr.adoc

```diff
@@ -8,11 +8,10 @@
 
 The following procedure creates a `UserDefinedNetwork` CR that is namespace scoped. Based upon your use case, create your request by using either the `my-layer-two-udn.yaml` example for a `Layer2` topology type or the `my-layer-three-udn.yaml` example for a `Layer3` topology type.
 
-//We won't have these pieces till GA in 4.18.
-//[NOTE]
-//====
-//If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`?
-//====
+.Perquisites
+
+* You have logged in with `cluster-admin` privileges, or you have `view` and `edit` role-based access control (RBAC).
```

Comment:
we can squeeze this somehow here `admin or view & edit roles in the subject namespace.` ?

Reply:
The only reason I didn't add the namespace specificity is because the first step of creating a UDN is to create a namespace so I thought we were covered. Also, all of those roles would be able to do that in the namespace.